### PR TITLE
libraries-bom: gax 1.67.0

### PR DESCRIPTION
**Pull request by igorbernstein2**

### CODE REVIEW

File: boms/cloud-oss-bom/pom.xml [hunk 1]
```diff
@@ -52,8 +52,8 @@
     <protobuf.version>3.17.3</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
-    <gax.version>1.66.0</gax.version>
-    <gax.httpjson.version>0.83.0</gax.httpjson.version>
+    <gax.version>1.67.0</gax.version>
+    <gax.httpjson.version>0.84.0</gax.httpjson.version>
```
Comment (meltsufin) [inline boms/cloud-oss-bom/pom.xml:56]: This is not in Maven Central yet.

Comment (igorbernstein2) [reply boms/cloud-oss-bom/pom.xml:56]: Looks like the kokoro job failed:
http://sponge/4d59a9d5-e7af-429c-b234-da0188f6f7d9

I'm asking someone from yoshi to take a look

**Comment by elharo** [comment]
Is there a reason you need this in advance of the next google-cloud-bom release?

**Comment by suztomo** [comment]
Now the Libraries BOM has gax:2.3.0.

**Pull request closed without merging**